### PR TITLE
artificial mica line

### PR DESCRIPTION
**Pull request by GlodBlock**
it is always an ass to find mica ore no matter when you in which tier. the only mica vein is in ow and certain t8 planet. this line provide a substitute for insulator foil when you don't want place a mbm in ow before you go to t8.

### CODE REVIEW

File: src/main/java/com/github/bartimaeusnek/bartworks/common/loaders/ArtificialMicaLine.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        //Mg + O = 2MgO
+        GT_Values.RA.addChemicalRecipe(Materials.Magnesium.getDust(1), null, Materials.Oxygen.getGas(1000), null, Materials.Magnesia.getDust(2), 40, 8);
+        //Si + 6HF = 7H2SiF6
+        GT_Values.RA.addChemicalRecipe(Materials.Silicon.getDust(1), null, Materials.HydrofluoricAcid.getFluid(6000), WerkstoffLoader.HexafluorosilicicAcid.getFluidOrGas(7000), null, 400, 120);
+        //K + Cl = 2KCl
+        GT_Values.RA.addChemicalRecipe(Materials.Potassium.getDust(1), GT_Utility.getIntegratedCircuit(2), Materials.Chlorine.getGas(1000), null, Materials.RockSalt.getDust(2), 20, 8);
+        //2KCl + H2SiF6 = 2HCl + K2SiF6
+        GT_Values.RA.addMixerRecipe(Materials.RockSalt.getDust(2), null, null, null, WerkstoffLoader.HexafluorosilicicAcid.getFluidOrGas(1000), Materials.HydrochloricAcid.getGas(2000), WerkstoffLoader.Potassiumfluorosilicate.get(OrePrefixes.dust, 1), 20, 8);
+        //2Al + 3O = Al2O3
+        GT_Values.RA.addChemicalRecipe(Materials.Aluminium.getDust(2), GT_Utility.getIntegratedCircuit(5), Materials.Oxygen.getGas(3000), null, WerkstoffLoader.Alumina.get(OrePrefixes.dust, 1), 10, 8);
```
Comment (Glease) [inline src/main/java/com/github/bartimaeusnek/bartworks/common/loaders/ArtificialMicaLine.java]: Isn't there a recipe for alumina already? Not sure if it has the same ratio though.

Comment (GlodBlock) [reply src/main/java/com/github/bartimaeusnek/bartworks/common/loaders/ArtificialMicaLine.java]: alumina is only obtainable from ore

Comment (Glease) [reply src/main/java/com/github/bartimaeusnek/bartworks/common/loaders/ArtificialMicaLine.java]: No, it's obtainable via (Al + Na) -> (NaAlO2) -> Al2O3 chain.

Comment (GlodBlock) [reply src/main/java/com/github/bartimaeusnek/bartworks/common/loaders/ArtificialMicaLine.java]: based

Comment (Glease) [reply src/main/java/com/github/bartimaeusnek/bartworks/common/loaders/ArtificialMicaLine.java]: I'd say just remove this recipe, or skip this step in this chain. You are bypassing the sodium hydroxide requirement of the Aluminium dust -> NaAlO2 -> Al2O3 -> Al smelting chain.

File: src/main/java/com/github/bartimaeusnek/bartworks/common/loaders/ArtificialMicaLine.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        //55Quartz Dust + 20K2SiF6 + 12Al2O3 + 4K2CO3 = 91Raw Fluorophlogopite Dust
+        GT_Values.RA.addMixerRecipe(Materials.QuartzSand.getDust(55), WerkstoffLoader.Potassiumfluorosilicate.get(OrePrefixes.dust, 20), WerkstoffLoader.Alumina.get(OrePrefixes.dust, 12), WerkstoffLoader.PotassiumCarbonate.get(OrePrefixes.dust, 4), null, GT_Utility.getIntegratedCircuit(4), null, null, null, null, null, WerkstoffLoader.RawFluorophlogopite.get(OrePrefixes.dust, 64), WerkstoffLoader.RawFluorophlogopite.get(OrePrefixes.dust, 27), null, null, 400, 120);
+        //55Quartz Dust + 20K2SiF6 + 12Al2O3 + 4K2CO3 = 91Raw Fluorophlogopite Dust
+        GT_Values.RA.addMixerRecipe(Materials.QuartzSand.getDust(55), WerkstoffLoader.Potassiumfluorosilicate.get(OrePrefixes.dust, 20), WerkstoffLoader.Alumina.get(OrePrefixes.dust, 12), WerkstoffLoader.PotassiumCarbonate.get(OrePrefixes.dust, 4), null, GT_Utility.getIntegratedCircuit(4), null, null, null, null, null, WerkstoffLoader.RawFluorophlogopite.get(OrePrefixes.dust, 64), WerkstoffLoader.RawFluorophlogopite.get(OrePrefixes.dust, 27), null, null, 400, 120);
```
Comment (Glease) [inline src/main/java/com/github/bartimaeusnek/bartworks/common/loaders/ArtificialMicaLine.java]: L33 and L35 looks identical.

File: src/main/java/com/github/bartimaeusnek/bartworks/common/loaders/ArtificialMicaLine.java [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+        //62Certus Quartz Dust + 10K2SiF6 + 12Al2O3 + 4K2CO3 = 91Raw Fluorophlogopite Dust
+        GT_Values.RA.addMixerRecipe(Materials.CertusQuartz.getDust(62), WerkstoffLoader.Potassiumfluorosilicate.get(OrePrefixes.dust, 10), WerkstoffLoader.Alumina.get(OrePrefixes.dust, 12), WerkstoffLoader.PotassiumCarbonate.get(OrePrefixes.dust, 4), null, GT_Utility.getIntegratedCircuit(4), null, null, null, null, null, WerkstoffLoader.RawFluorophlogopite.get(OrePrefixes.dust, 64), WerkstoffLoader.RawFluorophlogopite.get(OrePrefixes.dust, 27), null, null, 600, 120);
+        //MgO(s) = MgO(l)
+        GT_Values.RA.addFluidExtractionRecipe(Materials.Magnesia.getDust(1), null, Materials.Magnesia.getMolten(144), 0, 20, 120);
+        //27Raw Fluorophlogopite Dust + 432MgO(l) = 4464Fluorophlogopite(l)
+        GT_Values.RA.addBlastRecipe(WerkstoffLoader.RawFluorophlogopite.get(OrePrefixes.dust, 27), null, BW_GT_MaterialReference.Magnesia.getMolten(432), WerkstoffLoader.HotFluorophlogopite.getFluidOrGas(4608), Materials.Glass.getDust(1), null, 600, 480, 1700);
```
Comment (Glease) [inline src/main/java/com/github/bartimaeusnek/bartworks/common/loaders/ArtificialMicaLine.java:42]: 4608 (code) or 4464 (comment)?

**Review by Glease — APPROVED** [review]
Sorry for all the back and forth. It's slightly hard to understand what each recipe is given how long each line is.

---
You are just really good at inventing weird numbers, but it looks ok for now.

**Comment by Pxx500** [comment]
@GlodBlock can you provide screens of the recipes?

**Comment by GlodBlock** [comment]
image
image
image
image
image
image
image
image
image

**Pull request merged**